### ISSUE TITLE
Get recent version for pygalmesh for Windows on conda

### ISSUE THREAD

**Issue by stefsmeets** [hpgem/nanomesh#21]
`pygalmesh` is hosted on conda: https://anaconda.org/conda-forge/pygalmesh

However, the Windows version lags behind on the Linux / OSX versions. It would be nice if the Windows binaries would also be available, because pygalmesh is tricky to build on Windows otherwise for inexperienced users.

**Comment by stefsmeets** [comment]
Depends on #5

**Comment by stefsmeets** [comment]
No longer needed with #124
